noir-projects/noir-contracts/contracts/benchmarking_contract/src/main.nr

```diff
@@ -52,4 +52,10 @@ pub contract Benchmarking {
     fn broadcast(owner: AztecAddress) {
         context.emit_public_log(storage.balances.at(owner).read());
     }
+
+    // Does a bunch of heavy compute
+    #[public]
+    fn sha256_hash_2048(data: [u8; 2048]) -> [u8; 32] {
+        std::hash::sha256(data)
```

Comment:
I guess this is ok if you want just any heavy compute, but looks like Tom has been migrating things: https://github.com/AztecProtocol/aztec-packages/pull/11696 to another sha?